documentation/modules/con-securing-client-authentication.adoc

```diff
@@ -112,3 +112,30 @@ Decoding the generated password:
 ----
 echo "Z2VuZXJhdGVkcGFzc3dvcmQ=" | base64 --decode
 ----
+
+=== Providing custom password
+
+When the user is created, Strimzi will normally generate a new random password.
+In case you want to use your own password instead of the one generated by Strimzi, you can create a secret with the password and reference it in the `KafkaUser` resource:
```

Comment:
```suggestion
You can use your own password instead of the one generated by Strimzi. To do so, create a secret with the password and reference it in the `KafkaUser` resource:
```